ruma-push-gateway-api/src/push/send_event_notification/v1.rs

```diff
@@ -0,0 +1,371 @@
+//! [POST /_matrix/push/v1/notify](https://matrix.org/docs/spec/push_gateway/r0.1.1#post-matrix-push-v1-notify)
+
+use js_int::UInt;
+use ruma_api::{ruma_api, Outgoing};
+pub use ruma_common::push::{PusherData, Tweak};
```

Comment:
There should be a single location these are imported from.

```suggestion
use ruma_common::push::{PusherData, Tweak};
```